extensions/cornerstone/src/utils/findNearbyToolData.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Finds tool nearby event position triggered.
+ *
+ * @param {Object} commandsManager mannager of commands
+ * @param {Object} event that has being triggered
+ * @returns cs toolData or undefined if not found.
+ */
+export const findNearbyToolData = (commandsManager, evt) => {
+  if (!evt?.detail) {
+    return;
+  }
+  const { element, currentPoints } = evt.detail;
+  return commandsManager.runCommand(
+    'getNearbyToolData',
+    {
+      element,
+      canvasCoordinates: currentPoints?.canvas,
+    },
+    'CORNERSTONE'
```

Comment:
We need to find a way to set the context for the viewer type actually active at the moment - not something you need to worry about, and having this extracted is great because it means there is a single spot to set this.

Reply:
We don't need to, I switched it to look into all context if not specified, probably should work if there is no duplicated name